lib/task/import/csvAccessionImportTask.class.php

```diff
@@ -416,18 +416,17 @@ public function execute($arguments = [], $options = [])
                     $self->status['resourceTypes'] = self::refreshTaxonomyTerms(QubitTaxonomy::ACCESSION_RESOURCE_TYPE_ID);
                 }
 
-                $this->setObjectPropertyToTermIdLookedUpFromTermNameArray(
-                    $self,
-                    'resourceTypeId',
+                $self->object->resourceTypeId = $self->translateNameToTermId(
                     'resource type',
                     $data,
+                    [],
```

Comment:
The third argument passed to `translateNameToTermId()` is always an empty array, so it seems like an unnecessary parameter, and the method logic can be simplified if it's removed.